src/components/calcite-handle/calcite-handle.tsx

```diff
@@ -0,0 +1,95 @@
+import { Component, Element, Event, EventEmitter, Prop, h } from "@stencil/core";
+import { CSS, ICONS } from "./resources";
+
+@Component({
+  tag: "calcite-handle",
+  styleUrl: "calcite-handle.scss",
+  shadow: true
+})
+export class CalciteHandle {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * @internal - stores the activated state of the drag handle.
+   */
+  @Prop({ mutable: true, reflect: true }) activated = false;
+
+  /**
+   * Value for the button title attribute
+   */
+  @Prop({ mutable: true, reflect: true }) textTitle = "handle";
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteHandleElement;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Emmitted when the the handle is activated and the up or down arrow key is pressed.
+   * @event calciteHandleNudge
+   */
+  @Event() calciteHandleNudge: EventEmitter;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  handleKeyDown = (event: KeyboardEvent) => {
+    switch (event.key) {
+      case " ":
+        this.activated = !this.activated;
+        break;
+      case "ArrowUp":
+      case "ArrowDown":
+        if (!this.activated) {
+          return;
+        }
+        const direction = event.key.toLowerCase().replace("arrow", "");
+        this.calciteHandleNudge.emit({ handle: this.el, direction });
+        break;
+    }
+  };
+
+  handleBlur = () => {
+    this.activated = false;
+  };
+
+  handleClick = (event) => {
+    event.stopPropagation();
+  };
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  render() {
+    return (
+      <button
+        class={{ [CSS.handle]: true, [CSS.handleActivated]: this.activated }}
```

Comment:
question: are these classes applied to the host or in the shadowDom? If on the host, can we move this inside the shadow dom so they aren't present on the host for a user to modify.

Reply:
shadow dom.